stdlib/std.ncl

```diff
@@ -237,16 +237,17 @@
         if length == 0 then
           acc
         else
-          let rec go = fun acc n =>
-            if n == length then
-              acc
-            else
-              let next_acc =
-                %elem_at% array n
-                |> f acc
-              in
-              go next_acc (n + 1)
-              |> %seq% next_acc
+          let rec go =
+            fun acc n =>
+              if n == length then
+                acc
+              else
+                let next_acc =
+                  %elem_at% array n
+                  |> f acc
+                in
+                go next_acc (n + 1)
+                |> %seq% next_acc
```

Comment:
This is strange, it seems this reverts back to an old formatting style.

Reply:
The problem is that the Nickel flake devshell still has an old version of topiary 😅